lib/Collection/QueryType/Handler/Traits/BitBagEnabledTrait.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Layouts\Sylius\BitBag\Collection\QueryType\Handler\Traits;
+
+use Doctrine\ORM\QueryBuilder;
+
+use function count;
+
+trait BitBagEnabledTrait
+{
+    /**
+     * Builds the criteria for filtering only enabled entities.
+     */
+    private function addBitBagEnabledCriterion(QueryBuilder $queryBuilder): void
+    {
+        $x = count($queryBuilder->getRootAliases()) > 0
```

Comment:
Can we call this somewhere else other than `$x`? `$field` or something else similar?